src/components/Header/index.jsx

```diff
@@ -1,46 +1,57 @@
-import React, { useState } from 'react';
-import { useSelector } from 'react-redux';
-import { Link } from 'gatsby';
-import styled from 'styled-components';
-import './index.scss';
-import Sidebar from './Sidebar';
+import React, { useState } from "react"
+import { useSelector } from "react-redux"
+import { Link } from "gatsby"
+import styled from "styled-components"
+import "./index.scss"
+import Sidebar from "./Sidebar"
 
 const Logo = styled.img`
   width: 75px;
   height: 75px;
   padding-top: 30px;
-`;
```

Comment:
Why are you removing semi-colons?

Reply:
might be an issue with prettier.

Reply:
Yeah, disable `Prettier` for now and let ESLint handle the rest. We'll reconfigure Prettier with ESLint later as we go.